cypress/integration/form-builder/validation.test.js

```diff
@@ -0,0 +1,102 @@
+import {nanoid} from 'nanoid'
+import client from '../../helpers/sanityClientSetUp'
+
+const docId = `drafts.${nanoid()}`
+
+const doc = {
+  _id: docId,
+  _type: 'validationCI',
+  errorString: '',
+  warningString: '',
+  infoString: '',
+}
+
+/** Each fields validation rule is configured with min(5) */
+const VALID_VALUE = 'String longer than 5'
+const INVALID_VALUE = 'abc'
+
+async function patchDocument(payload) {
+  return client.patch(docId).set(payload).commit()
```

Comment:
Cool trick: If you use async visibility here it'll return a lot quicker :)
This tells the backend to not wait for the document to be "indexed" before returning, but you'll still be able to navigate to it in the studio and such!

```suggestion
  return client.patch(docId).set(payload).commit({visibility: 'async'})
```

Reply:
Thanks! Will add 👍